supabase/migrations/20230923153903_new_permit.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS
+  new_permits (
+    id serial primary key,
+    created_at timestamptz not null,
+    updated_at timestamptz not null,
+    network smallserial not null,
+    token varchar(42) not null,
+    amount numeric not null,
+    nonce numeric not null,
+    deadline numeric not null,
+    beneficiary varchar(42) not null,
+    owner varchar(42) not null,
+    signature varchar(132) not null
+  );
+
+INSERT INTO
+  new_permits (
+    id,
+    created_at,
+    updated_at,
```

Comment:
```suggestion
    created,
    updated,
```

Reply:
At seems implied. I think my suggestion is more clean

Reply:
> At seems implied. I think my suggestion is more clean

I just added the `_at` because that is what almost of all the tables use.. so i wanted to be consistent

Reply:
The new tables in all the specs I removed those suffixes

Reply:
I've remove all _at suffix